gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/ComplianceConfigurationKeys.java

```diff
@@ -33,6 +33,16 @@
   public static final String PARTITION_NAME = COMPLIANCE_PREFIX + ".partition.name";
   public static final int TIME_STAMP_LENGTH = 13;
   public static final String DBNAME_SEPARATOR = "__";
+  public static final String NUM_ROWS = "numRows";
+  public static final String RAW_DATA_SIZE = "rawDataSize";
```

Comment:
Wouldn't it be better to prefix this with original, or originalPartition to differentiate it from the purgedPartition (and introduce an additional property to hold the rows in the purgedPartition)?